packages/components/organisms/f-form/src/components/Form.vue

```diff
@@ -1,46 +1,103 @@
 <template>
-    <div
+    <form
```

Comment:
Let me know when you're free as would be good to go over this component – we played around with the idea of having a generic form component when we built registration, but for one reason or another decided against it. WOuld be good to understand the reasons for building one now, as guessing it's for use in checkout?